tests/multiprocess_gpu_test.py

```diff
@@ -209,5 +289,275 @@ def test_gpu_multi_node_initialize_and_psum(self):
     self.assertEqual(y[0], jax.device_count())
     print(y)
 
+  def test_pjit_gda_single_output(self):
+    jax_multiprocess_init()
+    with jtu.create_global_mesh((8, 2), ("x", "y")) as global_mesh:
+      global_input_shape = (16, 2)
+      mesh_axes = experimental.PartitionSpec("x", "y")
+      global_input_data = np.arange(
+          util.prod(global_input_shape)).reshape(global_input_shape)
+
+      def cb(index):
+        return global_input_data[index]
+
+      gda_obj = global_device_array.GlobalDeviceArray.from_callback(
+          global_input_shape, global_mesh, mesh_axes, cb)
+
+      jax.config.update('jax_parallel_functions_output_gda', True)
```

Comment:
Use the context manager. This is unsafe as you never set it to False.

Or better yet, set this once in `setUp()` and set it to the original value in `tearDown()`

Reply:
used context manager